lib/suma/api/me.rb

```diff
@@ -29,16 +29,28 @@ class Suma::API::Me < Suma::API::V1
       optional :address, type: JSON do
         use :address
       end
+      optional :organization_name, type: String, allow_blank: false
```

Comment:
What does it mean for a member to update their organization at any time like this?